### ISSUE TITLE
object and exact can't be called when exact_by_default=true

### ISSUE THREAD

**Issue by RohanHart** [nvie/decoders#436]
The error is:

Cannot call object with object literal bound to mapping because:
 • property x is missing in object type [1] but exists in object literal [2].
 • property y is missing in object type [1] but exists in object literal [2].
```
 [2] 18│ const pointDecoder = object({
     19│     x: number,
     20│     y: number,
     21│ });

     node_modules/decoders/object.js.flow
 [1] 78│ export function object<O: { +[field: string]: AnyDecoder }>(mapping: O): Decoder<$ObjMap<O, $DecoderType>> {
```

**Comment by nvie** [comment]
Thanks for reporting this issue. I will check this out soon, and think about the best way to resolve this.

**Comment by nvie** [comment]
This should be fixed and available in [v1.18.0](https://github.com/nvie/decoders/releases/tag/v1.18.0) now. Thanks for opening this issue!
